### ISSUE TITLE
FFmpegFrameRecorder encryption

### ISSUE THREAD

**Issue by huteh** [bytedeco/javacv#1131]
Dear Sir/Mdm,

I understand that FFmpeg is able to perform AES encryption.
Could I know how to implement this in Javacv? 
Have tried looking through the source code and forums but to no avail. 

Thanks

**Comment by saudet** [comment]
Yes, should be possible with the right protocol option: https://ffmpeg.org/ffmpeg-protocols.html

**Comment by huteh** [comment]
Could I kindly check on how to call the protocol/encryption function using FFmpegFrameRecorder class?

**Comment by saudet** [comment]
If the documentation is accurate, it looks like instead of just the URL we should pass "aes:URL".

**Comment by huteh** [comment]
Hi Saudet, thanks for the help. Can I check if there is any way for me to convert the frame into byte[] so that it can be passed through the network? I am intending to do a server/client broadcasting program.

**Comment by saudet** [comment]
Sure, we can use an OutputStream instead of a URL on construction. If you do that though, FFmpeg's documentation doesn't say how to enable encryption though...
